msicreator/create_installer.py

```diff
@@ -0,0 +1,62 @@
+import os, sys, subprocess
+from pathlib import Path
+from shutil import copytree, copy2
+import json
+import createmsi
+
+print('First arg:', sys.argv[1])
+build_dir = sys.argv[1]
+src_dir = sys.argv[2]
+work_dir = build_dir + '/workdir'
+project_path = work_dir + '/msicreator/libo-project'
+
+def prepare_project_dir():
+    instdir = build_dir + '/instdir'
+    try:
+        print('Copying build files...')
+        copytree(instdir, project_path + '/main', dirs_exist_ok=True)
+        print('Copying build files finished.')
+    except FileExistsError as err:
+        print(err)
+
+def create_creator_json():
+    print('Generating json file for MSI Creator...')
+    lo_dictionary = {
+        "upgrade_guid": "6f05ed48-a735-4155-ab60-e4cc98455262",
+        "version": "7.4.2",
+        "product_name": "LibreOffice",
+        "manufacturer": "The Document Foundation",
+        "name": "LibreOffice",
+        "name_base": "libreoffice",
+        "comments": "testing a libo installer",
+        "installdir": "LibreOffice Test",
+        "license_file": "./License.rtf",
+        "major_upgrade": {
+            "AllowDowngrades": "yes",
+            "IgnoreRemoveFailure": "yes"
+        },
+        "parts": [
+            {
+                "id": "libreoffice",
+                "title": "The LibreOffice Suite",
+                "description": "This is a test for the LibreOffice installer",
+                "absent": "disallow",
+                "staged_dir": "main"
+            }
+        ]
+    }
+    print('Serializing json object...')
+    lo_object = json.dumps(lo_dictionary, indent=4)
+    print('Writing data fo file...')
+    with open(project_path + '/lo.json', 'w') as lo_json:
+        lo_json.write(lo_object)
+    copy2(src_dir + '/msicreator/License.rtf', project_path)
+    print('Done!!!')
```

Comment:
While these kinds of prints are useful during development, they are typically removed before merging. They definitely need to be removed before this will be accepted to upstream master.